### DISCUSSION TITLE
ARM Based Docker Support

### DISCUSSION THREAD

**Discussion by dave-bell** [fatfreecrm/fat_free_crm#1702] [Show and tell]
### **Note:** This isn't an actual defect.  This is a public service announcement.

<hr />

I created a docker build that will provide an image that will run on either Intel hosts (`amd64`) or raspbian ARM hosts (`arm32v7`).  I forked the project to [Gitlab](https://gitlab.com/dave.bell/fat_free_crm) so I could get the CI system to build and publish the resulting images.

You can pull using `registry.gitlab.com/dave.bell/fat_free_crm:latest`.  That image is phat one, on Intel hosts you will get the `amd64` image while on a raspberry pi you will get the `arm32v7` image.  You can see all the [images here](https://gitlab.com/dave.bell/fat_free_crm/container_registry).

The builds are based off the `multiarch-build` branch.  The Gitlab repo pulls from GitHub on a regular basis.  When new changes are pulled into that branch a CI run kicks off and new images are published to the container registry.  At the moment changes need to be manually pulled into the build branch.

**Comment by johnnyshields** [comment]
Curious as to what the use case is to run FFCRM on ARM?

**Comment by CloCkWeRX** [comment]
Possibly running on an rpi?
